util/src/main/java/io/kubernetes/client/informer/cache/function/IndexFunc.java

```diff
@@ -0,0 +1,9 @@
+package io.kubernetes.client.informer.cache.function;
+
+import java.util.List;
+
+@FunctionalInterface
+public interface IndexFunc<T> {
```

Comment:
Let's remove this and use `java.util.function` instead?

Reply:
done